Sources_v3/StreamChatUI/ChatChannel/MessageComposerInputAccessoryViewController.swift

```diff
@@ -2,35 +2,161 @@
 // Copyright © 2020 Stream.io Inc. All rights reserved.
 //
 
+import StreamChat
 import UIKit
 
-class MessageComposerInputAccessoryViewController: UIInputViewController {
-    var composerView = ChatChannelMessageComposerView<DefaultUIExtraData>(uiConfig: .default).withoutAutoresizingMaskConstraints
-
-    override func viewDidLoad() {
+open class MessageComposerInputAccessoryViewController<ExtraData: UIExtraDataTypes>: UIInputViewController,
```

Comment:
nit: This should also conform to `Customizable` and `AppearanceSetting` like all our components

Reply:
Yes, I've done it at first place but protocol methods are not really relevant cause it's some kind of controlling object not actually a UI component so I'm not sure.